Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA275 */
+
+#include<iostream.h>
+#include<conio.h>
+void main()
+{
+	int n;
+	clrscr();
+	cout<<"Enter a number : ";
+	cin>>n;
+	if(n%2==0)

```

Comment:
why used is this (n%2==0).?

Reply:
this is a if condition n%2 if remainder comes 0 its means it is even number else odd number.

Reply:
ok thanks